compiler/rustc_codegen_llvm/src/back/write.rs

```diff
@@ -853,6 +853,27 @@ fn create_section_with_flags_asm(section_name: &str, section_flags: &str, data:
     asm
 }
 
+pub(crate) fn bitcode_section_name(is_apple: bool, is_aix: bool) -> &'static str {
```

Comment:
I think it would be a bit cleaner to accept cgcx in this function and make it do the is_apple / is_aix check itself, so we don't have to pass around the two separate parameters.

Reply:
sure